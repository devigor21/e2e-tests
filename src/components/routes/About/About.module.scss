.wrapper {
  margin: 5rem 2rem;
  color: rgb(30, 34, 47)
}
