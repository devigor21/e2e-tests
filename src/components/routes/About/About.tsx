import React, { FC } from 'react';

import classes from './About.module.scss';

const About: FC = () => (
  <div className={classes.wrapper}>
    <h1 style={{ color: 'rgb(40, 41, 51)' }}>This is website for homeworks</h1>
    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit.
      Deleniti dignissimos quos ab animi dolores autem!
      Exercitationem architecto, laborum qui veniam,
      sint ipsam possimus facilis iure eius
      dignissimos tempora facere repellendus
      animi ratione reiciendis vero odio nostrum
      dolores cupiditate odit dolorem. Unde magni
      possimus qui, ipsum culpa soluta aut voluptas
      ullam, ipsam quo quos, repudiandae numquam
      id ratione deleniti. Labore corrupti veritatis
      ipsum velit voluptatem vel quod, doloribus
      quaerat laboriosam ut. Laborum nesciunt
      distinctio esse deserunt aperiam nam?
      Quae vero veritatis explicabo, at ab recusandae
      numquam adipisci labore nam? Fugit ut, rem non
      facere pariatur possimus ipsam ullam ea nostrum
      delectus quaerat ratione eos cum tempora,
      saepe molestias distinctio quis fugiat repudiandae.
    </p>
  </div>
);

export default About;
