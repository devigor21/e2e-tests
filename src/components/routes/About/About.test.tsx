import React from 'react';
import { render, screen } from '@testing-library/react';

import About from '.';

describe('About component', () => {
  it('should not change', async () => {
    const about = render(<About />);
    expect(about).toMatchSnapshot();
  });
  it('should have different header color', () => {
    render(<About />);
    expect(screen.getByRole('heading')).toHaveStyle('color:rgb(40, 41, 51)');
  });
  it('should have correct header', () => {
    render(<About />);
    expect(screen.getByText(/This is website for homeworks/i)).toBeInTheDocument();
  });
});
