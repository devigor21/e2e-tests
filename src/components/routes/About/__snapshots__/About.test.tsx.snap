// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should not change 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="wrapper"
      >
        <h1
          style="color: rgb(40, 41, 51);"
        >
          This is website for homeworks
        </h1>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti dignissimos quos ab animi dolores autem! Exercitationem architecto, laborum qui veniam, sint ipsam possimus facilis iure eius dignissimos tempora facere repellendus animi ratione reiciendis vero odio nostrum dolores cupiditate odit dolorem. Unde magni possimus qui, ipsum culpa soluta aut voluptas ullam, ipsam quo quos, repudiandae numquam id ratione deleniti. Labore corrupti veritatis ipsum velit voluptatem vel quod, doloribus quaerat laboriosam ut. Laborum nesciunt distinctio esse deserunt aperiam nam? Quae vero veritatis explicabo, at ab recusandae numquam adipisci labore nam? Fugit ut, rem non facere pariatur possimus ipsam ullam ea nostrum delectus quaerat ratione eos cum tempora, saepe molestias distinctio quis fugiat repudiandae.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="wrapper"
    >
      <h1
        style="color: rgb(40, 41, 51);"
      >
        This is website for homeworks
      </h1>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti dignissimos quos ab animi dolores autem! Exercitationem architecto, laborum qui veniam, sint ipsam possimus facilis iure eius dignissimos tempora facere repellendus animi ratione reiciendis vero odio nostrum dolores cupiditate odit dolorem. Unde magni possimus qui, ipsum culpa soluta aut voluptas ullam, ipsam quo quos, repudiandae numquam id ratione deleniti. Labore corrupti veritatis ipsum velit voluptatem vel quod, doloribus quaerat laboriosam ut. Laborum nesciunt distinctio esse deserunt aperiam nam? Quae vero veritatis explicabo, at ab recusandae numquam adipisci labore nam? Fugit ut, rem non facere pariatur possimus ipsam ullam ea nostrum delectus quaerat ratione eos cum tempora, saepe molestias distinctio quis fugiat repudiandae.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
